en/docs/chapter_searching/index.md

```diff
@@ -4,6 +4,6 @@
 
 !!! abstract
 
-    Searching is an unknown adventure, where we may need to traverse every corner of a mysterious space, or perhaps quickly pinpoint our target.
+    Searching is an adventure into the unknown; where we may need to traverse every corner of a mysterious space, or perhaps we’ll quickly pinpoint our target.
```

Comment:
I would prefer using "locate" instead of "pinpoint" here as it sounds more common to be used in computer world.